RieGeo.tex

```diff
@@ -779,7 +790,7 @@ \chapter{Riemann’sche Metriken}
 
 \section{Definition einer Riemann’schen Metrik und Struktur}
 \newcommand{\asp}{\langle\cdot,\cdot\rangle} % Abstraktes Skalarprodukt
-\newcommand{\ksp}[2]{\langle#1,#2\rangle} % Konkretes Skalarprodukt mit zwei Parameter
+\newcommand{\ksp}[2]{\left\langle#1,#2\right\rangle} % Konkretes Skalarprodukt mit zwei Parameter
 \newcommand{\lsp}[2]{\left\langle#1,#2\right\rangle} % großes Skalarprodukt mit zwei Parameter
```

Comment:
Jetzt ist `\ksp` das gleiche wie `\lsp`. Warum nicht dort, wo nötig, `\ksp` in `\lsp` ändern?